monitoring/monitorlib/messagesigning/uft.py

```diff
@@ -0,0 +1,7 @@
+def get_x_utm_jws_header(cert_url):
+    return 'alg="{}", typ="{}", kid="{}", x5u="{}"'.format(
```

Comment:
It seems odd to fill format string placeholders with constants; why not simply:

```suggestion
    return 'alg="RS256", typ="JOSE", kid="mock_uss_priv_key", x5u="{}"'.format(cert_url)
```